### PR TITLE
hmm,more like that :d

### PR DESCRIPTION
**Pull request by ghost**
More like that and players can watch more easy message :100:

### CODE REVIEW

File: resources/deu.ini [hunk 1]
```diff
@@ -1,8 +1,13 @@
 Language = 'Deutsch'
+
 [General]
+
```
Comment (SalmonGER) [inline resources/deu.ini]: Can you remove the lines after [General] [MySQL] and [Player]?

**Comment by SalmonDE** [comment]
Can you remove the lines after [General] [MySQL] and [Player]?

File: resources/eng.ini [hunk 2]
```diff
@@ -11,13 +16,19 @@ CreateDatabaseSuccess = 'Database created! '
 CreateDatabaseFailure = 'Could not create database: '
 NotFoundDatabase = 'Database not found, creating new ...'
 DataSavingError = 'Error while saving data of {player}: {error}'
-; These message will most likely be sent to players!
+
+
+# These message will most likely be sent to players!
```
Comment (SalmonDE) [inline resources/eng.ini]: This is an error

File: resources/eng.ini [hunk 3]
```diff
@@ -29,8 +40,11 @@ FloatingStatExists = 'FloatingStats {name} exists already!'
 FloatingStatNotExists = 'FloatingStats {name} does not exist!'
 FloatingStatCreateSuccess = 'FloatingStats {name} created!'
 FloatingStatRemoveSuccess = 'Successfully removed FloatingStats {name}!'
-; These settings will be used for XBoxAuthenticated and Online
-; (Up to 10 letters!)
+
+
+# These settings will be used for XBoxAuthenticated and Online
+# (Up to 10 letters!)
```
Comment (SalmonDE) [inline resources/eng.ini]: It must be ;

**Comment by SalmonDE** [comment]
Comments must begin with ";". Please fix it

**Comment by ghost** [comment]
@SalmonGER Why you want to delete [General] [MySQL] and [Player] ?

More like that because player know for what is this message ;)

**Comment by SalmonDE** [comment]
@RoofyManStar I mean make it together with the other stuff
Like:
[MySQL]
Settings

not:
[MySQL]

Settings

**Comment by ghost** [comment]
@SalmonGER ready

**Comment by SalmonDE** [comment]
@RoofyManStar :+1:

**Pull request merged**